lib/rafka/consumer.rb

```diff
@@ -108,6 +116,23 @@ def consume(timeout=5)
       end
     end
 
+    # Commit offsets for the given messages.
+    #
+    # If more than one messages refer to the same topic/partition pair,
+    # only the largest offset amongst them is committed.
+    #
+    # @note This is non-blocking operation; a successful reply means
+    #   offsets are received by the server and will eventually be committed
+    #   to Kafka.
+    #
+    # @param [Array<Message>] the messages for which to commit offsets
+    #
+    # @raise [ConsumeError] if there was any error commiting offsets
+    #
+    # @return [Boolean] whether offsets were committed successfully
```

Comment:
Not sure about the return value. We probably don't know if the offsets are actually commited to kafka, especially since we plan to switch the implemenation to the `store_offsets` librdkafka API.

Reply:
I've updated this to return the number of offsets pairs that were successfully sent to rafka.

Reply:
True, that would be nice! We just have to state that we are not sure that those offsets are actually committed to kafka. This should not be a problem, we already know that kafka guarrantees `at-least-once` semantics and the consumer code should already handle multiple deliveries.